web/src/main/resources/org/akaza/openclinica/i18n/words.properties

```diff
@@ -579,7 +579,7 @@ confirmation_screen = Confirmation Screen
 
 congratulations_your_spreadsheet = <b>Congratulations!</b> Your spreadsheet generated no errors.
 
-congratulations_your_spreadsheet_no_errors = <b>Congratulations!</b>  Your spreadsheet generated no errors.<br/><b>Please review it.<br> You can save it to database by clicking "Continue".</b>
```

Comment:
why are you removing these html tags? If these are causing security issues, we need to find another approach to preserve the formatting of this text.

Reply:
Hey Shiva. Yes they are changes due to the security fixes for the new CSP policies. Malcolm looked it over and he thinks the changes look better than the original.